fedimint-client-module/src/module/init.rs

```diff
@@ -239,4 +239,17 @@ pub trait ClientModuleInit: ModuleInit + Sized {
     fn get_database_migrations(&self) -> BTreeMap<DatabaseVersion, ClientMigrationFn> {
         BTreeMap::new()
     }
+
+    /// Db prefixes used by the module
+    ///
+    /// If `Some` is returned, it should contain list of database
+    /// prefixes actually used by the module for it's keys.
+    ///
+    /// In (some subset of) non-production tests,
+    /// module database will be scanned for presence of keys
+    /// that do not belong to this list to verify integrity
+    /// of data and possibly catch any unforeseen bugs.
+    fn used_db_prefixes(&self) -> Option<BTreeSet<u8>> {
```

Comment:
I made in `Option`, just to not force it on 3rd party developers. At least yet. Also just checking our own keys is probably enough to ensure our abstractions are not playing any tricks.